### PR TITLE
Zib2020 HelpFromOthers

### PR DESCRIPTION
**Pull request by LuudSlagter**

### CODE REVIEW

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-01.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   </meta>
+   <status value="active"/>
+   <intent value="plan"/>
+   <category>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="243114000"/>
+         <display value="ondersteuning"/>
+      </coding>
+   </category>
```
Comment (Ashwin-nictiz) [inline ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-01.xml:15]: Error from the QA tool:

```
  -  error at CarePlan.category[0] (9, 14):
     Slicing cannot be evaluated: Could not match discriminator ([coding]) for slice CarePlan.category:helpFromOthers in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers - the discriminator [coding] does not have fixed value, binding or existence assertions
```

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-02.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+   </meta>
+   <status value="active"/>
+   <intent value="plan"/>
+   <category>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="243114000"/>
+         <display value="ondersteuning"/>
+      </coding>
+   </category>
```
Comment (Ashwin-nictiz) [inline ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-02.xml:15]: Error from the QA tool:

```
 -  error at CarePlan.category[0] (9, 14):
     Slicing cannot be evaluated: Could not match discriminator ([coding]) for slice CarePlan.category:helpFromOthers in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers - the discriminator [coding] does not have fixed value, binding or existence assertions
```

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-01.xml (hunk 1 above)
Comment (LuudSlagter) [reply ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-01.xml:15]: Error has been fixed, see https://github.com/Nictiz/Nictiz-R4-zib2020/pull/308.

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-02.xml (hunk 2 above)
Comment (LuudSlagter) [reply ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-HelpFromOthers-02.xml:15]: Error has been fixed, see https://github.com/Nictiz/Nictiz-R4-zib2020/pull/308.

**Pull request merged**